core/arch/arm/plat-imx/pm/psci.c

```diff
@@ -47,9 +47,22 @@
 #include <tee/entry_std.h>
 #include <tee/entry_fast.h>
 
+int psci_features(uint32_t psci_fid)
+{
+    switch (psci_fid) {
+#ifdef CFG_BOOT_SECONDARY_RQUEST
```

Comment:
Fix typo

Reply:
Maybe move this change into a specific commit.